CoSiExample.md

```diff
@@ -0,0 +1,109 @@
+Navigation: [DEDIS](https://github.com/dedis/doc/tree/master/README.md) ::
+[Cothority Template](../README.md) ::
+CoSi Explanation
+
+# CoSi Explanation
```

Comment:
This section seems to suggest that cosi should be used. Should use ftcosi as an example instead, or add a warning that says "you should use ftcosi if you assume fail-stop faults".

Reply:
Added a comment saying that ftCoSi is the successor. But for showing how a simple protocol works, CoSi is clearer, as in ftCoSi you have to register the protocol yourself.